### PR TITLE
added fix for VPN tunnel issue

### PR DESCRIPTION
**Pull request by mayankbharadwaj**
Purpose: 
 
VPN tunnels are not getting created for new account.

Changes:

-  modified iam.js

Testing:
 
Testing is completed.

### CODE REVIEW

**Review by bhargavapratik — APPROVED** [review]
LGTM

**Review by SonalOjha — APPROVED** [review]
LGTM

File: aws_promise/iam.js [hunk 1]
```diff
@@ -3,46 +3,41 @@
...
-    else {
-      let params = {
-          PolicyName: input.roleName+"Policy",
-          RoleName: input.roleName,
-          PolicyDocument: JSON.stringify(input.policyDocument)
+    } else {
+      params = {
+        PolicyName: `${input.roleName}Policy`,
+        RoleName: input.roleName,
+        PolicyDocument: JSON.stringify(input.policyDocument)
```
Comment (SonalOjha) [inline aws_promise/iam.js:38]: What if the policy document is an empty string?

Comment (mayankbharadwaj) [reply aws_promise/iam.js:38]: In managedVpcPreconfig we are not passing policyDocument. so it will not come under this condition anyways

Comment (SonalOjha) [reply aws_promise/iam.js:38]: But shall be addressed despite of that.

**Pull request merged**